hack/jenkins/test-image/Dockerfile

```diff
@@ -15,7 +15,7 @@
 # This file creates a build environment for building and running kubernetes
 # unit and integration tests
 
-FROM golang:1.5.3
+FROM golang:1.6
```

Comment:
Ah, didn't think about it that way. Of course the CI should be on the edge.
Good point :)